src/main/kotlin/no/nav/familie/ef/sak/mapper/TilkjentYtelseMapper.kt

```diff
@@ -38,7 +37,7 @@ fun TilkjentYtelseDTO.tilAndelerTilkjentYtelse(): List<AndelTilkjentYtelse> {
 
 fun TilkjentYtelse.tilDto(): TilkjentYtelseDTO {
     return TilkjentYtelseDTO(id = this.id,
```

Comment:
Hvem er det som skal bruke TilkjentYtelseDto? Frontend? Isåfall tror jeg behandlingId bør være UUID-en.